src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -15,11 +15,18 @@ export function fetchInstitutions({ institutionQuery, page }) {
       'X-Key-Inflection': 'camel'
     }
   })
-    .then(res => res.json())
+    .then(res => {
```

Comment:
could we use the [apiRequest helper ](https://github.com/department-of-veterans-affairs/vets-website/blob/dc5cd8642a10aa654a9fe54b53d55dccafce6acc/src/platform/utilities/api/index.js#L13)here?

Reply:
I'm pretty against using fetch abstractions. This one for example does a redirect if the response status is 401. 😱 That is a really invasive + obfuscated behavior.